modules/gdscript/gd_parser.cpp

```diff
@@ -2014,7 +2023,7 @@ void GDParser::_parse_block(BlockNode *p_block,bool p_static) {
 			} break;
 			default: {
 
-				Node *expression = _parse_and_reduce_expression(p_block,p_static,false,true);
+				Node *expression = _parse_and_reduce_expression(p_block,p_static,true,true);
```

Comment:
@akien-mga You think so? It is never true in the other parts of the code :laughing: 

Let's see if we break anything though...